queries.sql

```diff
@@ -1,3 +1,56 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth > '2015-12-31' and date_of_birth < '2020-01-01';
+SELECT * FROM animals WHERE neutered = true and escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
-SELECT * from animals WHERE name = 'Luna';
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN:
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+BEGIN;
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ SELECT * FROM animals;
+ ROLLBACK;
+  SELECT * FROM animals
+  DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+  SELECT * FROM animals;
+  SAVEPOINT SP1;
+  UPDATE animals
+SET weight_kg = weight_kg * -1;
+ SELECT * FROM animals;
+ROLLBACK TO SP1;
+  SELECT * FROM animals;
+  COMMIT;
```

Comment:
- [ ] this should be a new transaction and therefor it should start with the BEGIN keyword.